src/jsMain/kotlin/dev/bitspittle/firebase/externals/auth/Externals.kt

```diff
@@ -74,6 +74,9 @@ internal external interface User : UserInfo {
     val metadata: UserMetadata
     val providerData: Array<UserInfo>
     val refreshToken: String
+
+    // https://firebase.google.com/docs/reference/js/v8/firebase.User#getidtoken
```

Comment:
Nit: Let's link to the auth version of the docs, for consistency with the rest of the file

https://firebase.google.com/docs/reference/js/auth.user#usergetidtoken